app/assets/scripts/views/task-page.js

```diff
@@ -109,7 +111,10 @@ var TaskPage = React.createClass({
       }
     }
 
-    const geometry = geometryToFeature([this.props.task.data]);
+    const geometry = {
+      'type': 'FeatureCollection',
+      'features': [geometryToFeature([this.props.task.data])]
+    };
```

Comment:
@nbumbarger The map on the task page was not working because the map expects a `featureCollection` and we were passing a normal `feature`.
I added this quick fix just to make it work, but I'd like you to revisit this and make it abstract enough that handles the different situations.

The other function on `utils/features` is not able to handle the data format of the tasks (the `geometry` property.). Can we make this more abstract/ flexible?

Without this code it is throwing and error on line 68, of `display-map.js` because `feat.features` is undefined.